### ISSUE TITLE
Interpreting an increase in CN for the majority of the genome

### ISSUE THREAD

**Issue by haraldgrove** [BoevaLab/FREEC#80]
I've just analaysed a WGS dataset (matched tumor-normal) from a breast cancer sample and the output is telling me that most of the genome is noted as having an increase in copy numbers (CN=3). (see figure of Ratio*2 below)

I just wanted to know if there might be an issue with the normalisation and that the real result should have been mostly normal regions with some regions having a reduction in copy number instead? 

Thank you

![WGS_BB-T0040-DNA cnv](https://user-images.githubusercontent.com/7669954/103962380-6b72ff00-5189-11eb-835d-5e1e7c4de521.png)

**Comment by valeu** [comment]
Dear Harald, it depends on whether you used the [BAF] option or not. If you used it, then FREEC will likely assign CN=3 correctly. Did you look at the resulting BAF?

**Comment by haraldgrove** [comment]
I did not use any of the BAF options. I can try and add those and see what happens.

Thanks.

**Comment by valeu** [comment]
Yes, please share then the second graph with BAF values to be sure that it worked!

**Comment by haraldgrove** [comment]
I re-did the analysis with BAF, and the CN-values changed for several of the sections. I'd assume that this is then a more correct view of the CNVs.

![WGS_BB-T0040-DNA cnv](https://user-images.githubusercontent.com/7669954/111097552-5beea880-8574-11eb-8ac5-b243aece3c48.png)

**Comment by valeu** [comment]
You should also visualize the BAF info. Does it make sense?

**Comment by haraldgrove** [comment]
I'm not sure if I can say it makes sense or not. The BAF matches the other results, but I don't know how it would look if there was anything wrong.

![BB-T0040-DNA_recal bam_BAF txt](https://user-images.githubusercontent.com/7669954/111275855-32fc0f80-8669-11eb-8a53-b35c577d5b34.png)

**Comment by valeu** [comment]
It fits! But I will update the visualization script written 10 years ago to make things more readable.
